test/unit-tests/components/views/elements/AppTile-test.tsx

```diff
@@ -160,29 +149,26 @@ describe("AppTile", () => {
                 />
             </MatrixClientContext.Provider>,
         );
-        // Wait for RPS room 1 updates to fire
-        const rpsUpdated = waitForRps("r1");
-        dis.dispatch({
-            action: Action.ViewRoom,
-            room_id: "r1",
-        });
-        await rpsUpdated;
+        act(() =>
+            dis.dispatch({
+                action: Action.ViewRoom,
+                room_id: "r1",
+            }),
+        );
 
-        expect(renderResult.getByText("Example 1")).toBeInTheDocument();
+        await expect(renderResult.findByText("Example 1")).resolves.toBeInTheDocument();
         expect(ActiveWidgetStore.instance.isLive("1", "r1")).toBe(true);
 
-        const { container, asFragment } = renderResult;
-        expect(container.getElementsByClassName("mx_Spinner").length).toBeTruthy();
-        expect(asFragment()).toMatchSnapshot();
```

Comment:
Is removing this snapshot assertion intentional?

Reply:
Good spot